.github/workflows/cleanup-terraform.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: cleanup-terraform
+
+on:
+  pull_request:
+    branches: [main]
+    types: [closed]
+
+jobs:
+  cleanup-terraform:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint
+    # so that we can authenticate with AWS
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Terraform
+        uses: ./.github/actions/setup-terraform
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
+          terraform-iam-role-arn: ${{ secrets.TERRAFORM_IAM_ROLE_ARN }}
+          aws-account-id: ${{ secrets.AWS_ACCOUNT_ID }}
+
+      - name: Delete resources using Terraform
+        run: terraform destroy -auto-approve
```

Comment:
The [`destroy` command](https://developer.hashicorp.com/terraform/cli/commands/destroy) is a shortcut for deleting all resources in the current workspace, in this case the workspace for the PR that is being closed. The [`-auto-approve` flag](https://developer.hashicorp.com/terraform/cli/commands/apply#automatic-plan-mode) overrides Terraform's default behavior to indicate that we do not want Terraform to wait for `y`/`n` prompt input before applying changes.

Reply:
Yup, that's right!